### PR TITLE
add all JSON.stringify args to writeJSON function

### PR DESCRIPTION
**Pull request by adblanc**
Hello,

First of all, thanks for this whole project, I really enjoy it.
However, I found myself wanting to indent the json content of my file but couldn't using the `writeJSON` from the lib.
I guess this shouldn't add any regressions, please let me know what you think about this 😄

### CODE REVIEW

**Comment by irealva** [comment]
Thank you! This seems useful and your code works well.

My only request is would you be able to take a pass at the README? 

We would have to at the least:
*  update the inputs for `writeJSON` 
* ideally, keep the [usage example](https://github.com/githubocto/flat-postprocessing#writejson), just add an extra one that shows the additional parameters

File: tests/json-test.ts [hunk 1]
```diff
@@ -18,5 +19,16 @@ Deno.test("writes a json file", async () => {
     await writeJSON(jsonWritePath, data)
     const json = await readJSON(jsonWritePath)
 
+    assertObjectMatch(json, { prices: "$20", name: "table" });
+})
+
+Deno.test("writes a json file with space indent", async () => {
```
Comment (irealva) [inline tests/json-test.ts:25]: Thanks for adding a test!

**Comment by adblanc** [comment]
Thanks for the review, I had completely forgot the README part ! 😄 

I used `any` to type `replacer` in the README to simplify as it's an overload with either `(this: any, key: string, value: any) => any` or `(number | string)[] | null`

**Comment by irealva** [comment]
Thanks!

**Pull request merged**